src/blockers.ts

```diff
@@ -1,5 +1,5 @@
 // Maps building jsonpath to values mappings that will fail database constraints
-const strLength = (val:string, [min, max]:[number, number]) => val.length > max || val.length < min 
+const strLength = (val:string, [min, max]:[number, number]) => val?.length > max || val?.length < min
```

Comment:
Does this work?  I would think we want to make sure that we default to 0, otherwise this could have weird side effects...
```typescripts
const strLength = (val:string, [min, max]:[number, number]) => (val?.length || 0) > max || (val?.length || 0) < min

Reply:
That depends on you want it to return true or false in this case
![image](https://github.com/user-attachments/assets/44cc6497-c66b-4057-95e8-aa013d99b2d1)

Reply:
So how is this occurring?  I think in some cases, we simply ignore NULL/UNDEFINED, and in others, parse and fail with them... Depending on how it got here may make the difference